src/components/contract/ContractExecutor.tsx

```diff
@@ -58,10 +57,27 @@ const useContractFunctions = () => {
     }, [contractStore.contract.textContent]);
 };
 
+const getValue = (value: string, type?: string) => {
+    switch (type) {
+        case ParamType.Int16:
```

Comment:
Seems like we heavily rely on these sorts of string constants - what do you think about moving them over to `wavelet-client-js` so that we can reference them throughout Lens?

Reply:
This is particular case where all values were coming from the <input> elements as strings.

If we move this to wavelet-client-js then all inputter param values should be strings, which could be unintuitive. Alternatively, we could cater to both scenarios and check each param value to allow for both string and default type.

Reply:
Hmm, perhaps that would not have to be worried about then for this current PR. Let's leave it as it is for now.